### PR TITLE
more announcements

### PR DESCRIPTION
**Pull request by ericayin**

### CODE REVIEW

File: sage-ios/SAGE/SAGE/AnnouncementsTableViewCell.swift [hunk 1]
```diff
@@ -47,7 +47,15 @@ class AnnouncementsTableViewCell: UITableViewCell {
...
         self.announcementMessage.text = announcement.text
-        self.announcementTime.text = "5 days ago"
+        
+        var calendar: NSCalendar = NSCalendar.currentCalendar()
+        let date1 = calendar.startOfDayForDate(announcement.timeCreated!)
+        let date2 = calendar.startOfDayForDate(NSDate())
+        let flags = NSCalendarUnit.Day
+        let components = calendar.components(flags, fromDate: date1, toDate: date2, options: [])
+        var days = components.day
+        self.announcementTime.text = String(days) + " days ago"
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/AnnouncementsTableViewCell.swift]: can you say something like today if it was 0 days ago

File: sage-ios/SAGE/SAGE/SignUpTableViewController.swift [hunk 2]
```diff
@@ -74,7 +74,7 @@ class SignUpTableViewController: UITableViewController, UINavigationBarDelegate
     }
     
     func loadSchoolData() {
-        AnnouncementsOperations.loadSchools { (schoolDict) -> Void in
+        LoginHelper.loadSchools { (schoolDict) -> Void in
```
Comment (amillman) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: wait...it should be LoginOperations. Does this version of the code compile?

Comment (ericayin) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: yea i think i showed sameera and she just named it differently? @sameerav

File: sage-ios/SAGE/SAGE/AnnouncementsTableViewCell.swift (hunk 1 above)
Comment (ericayin) [inline sage-ios/SAGE/SAGE/AnnouncementsTableViewCell.swift]: would "today" be better cause i think its 0 days right now?

Comment (ericayin) [inline sage-ios/SAGE/SAGE/AnnouncementsTableViewCell.swift]: oh jk misread your comment

File: sage-ios/SAGE/SAGE/SignUpTableViewController.swift (hunk 2 above)
Comment (ericayin) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: oh wait nvm she renamed it

Comment (ericayin) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: @sameerav should i just take the whole thing out since theres no more loginhelper file

Comment (sameerav) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: yes please remove this file

Comment (ericayin) [inline sage-ios/SAGE/SAGE/SignUpTableViewController.swift]: LoginOperations doesnt have a loadSchools function so i just took the old one and put it in... is that ok

**Pull request merged**